core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -672,7 +672,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
   @Test
   def testFetchInvalidOffset(): Unit = {
     this.consumerConfig.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "none")
-    val consumer = createConsumer()
+    val consumer = createConsumer(configOverrides = this.consumerConfig)
```

Comment:
Nice fix!

Reply:
i think that `createConsumer()` basically does this anyways (it reads `this.consumerConfig` and passes it through, but i agree this is more explicit)